core/app/clients/uffizzi_core/http_request_decorator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module UffizziCore::HttpRequestDecorator
```

Comment:
We use it only for ContainerRegistry therefore `HttpRequestDecorator` is very common name for this. Maybe `ContainerRegistryHttpRequestDecorator`